x/evm/migrations/v4/migrate.go

```diff
@@ -27,24 +27,14 @@ func MigrateStore(
 		return err
 	}
 
-	chainCfgBz := cdc.MustMarshal(&params.ChainConfig)
-	extraEIPsBz := cdc.MustMarshal(&types.ExtraEIPs{EIPs: types.AvailableExtraEIPs})
+	params.ExtraEIPs.EIPs = types.AvailableExtraEIPs
 
-	store.Set(types.ParamStoreKeyEVMDenom, []byte(params.EvmDenom))
-	store.Set(types.ParamStoreKeyExtraEIPs, extraEIPsBz)
-	store.Set(types.ParamStoreKeyChainConfig, chainCfgBz)
-
-	if params.AllowUnprotectedTxs {
-		store.Set(types.ParamStoreKeyAllowUnprotectedTxs, []byte{0x01})
-	}
-
-	if params.EnableCall {
-		store.Set(types.ParamStoreKeyEnableCall, []byte{0x01})
+	bz, err := cdc.Marshal(&params)
```

Comment:
@GAtom22 the individual keys that we introduced in v0.21-rc1 need to be deleted as well btw so that we can handle the migration from `v11.0.0-rc1` and also from `v10` on Evmos

Reply:
@fedekunze don't we need these for the `GetLegacyParams` function which calls `GetParamSetIfExists`? I see these are used on the [`ParamSetPairs()`](https://github.com/evmos/ethermint/blob/6505f6ee8d531bdd2aee6781defd48901b967c55/x/evm/types/params.go#L90-L100) which is [called on the `GetParamSetIfExists` function ](https://github.com/cosmos/cosmos-sdk/blob/b73462f930da41539447dad322181efc324dedcb/x/params/types/subspace.go#L243-L250)

Reply:
yeah, that applies to v10. We need the remove the new keys on testnet which is on v11-rc1 (eg. `types.ParamStoreKeyEVMDenom`, etc)